notebook.ipynb

```diff
@@ -2,7 +2,15 @@
  "cells": [
```

Comment:
<p>Would prefer if the link to the license points to <a href="https://github.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/blob/main/LICENSE" target="_blank">https://github.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/blob/main/LICENSE</a> rather than <a href="https://github.com/alan-turing-institute/environmental-ds-book/blob/master/LICENSE.md" target="_blank">https://github.com/alan-turing-institute/environmental-ds-book/blob/master/LICENSE.md</a>, just in case the wider EDS license ever changes in the future, this notebook won't be accidentally re-licensed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>

Reply:
<p>@annefou, happy with this change?</p><p>Current wording seems to be what's generally used in EDS Book, just seeking confirmation.</p>

Reply:
We discussed this change and indeed the link to the license should point to the license file that is present in the notebook repository (here:  https://github.com/eds-book-gallery/67a1e320-7c47-4ea9-8df8-e868326bc90b/blob/main/LICENSE), rather than the EDSbook license file. I think we will need to update all the notebook repos accordingly. @acocac do you agree with this suggestion?

Thanks for raising this issue.

Reply:
Hi @acocac, thanks for reaching out. I agree we should point to the license file of the notebook repository. I have opened an issue to report this, https://github.com/alan-turing-institute/environmental-ds-book/issues/242. Thanks @weiji14.

Reply:
<p>Updated in latest review based updates to point to local LICENSE file.</p>

Reply:
Thank you.